ete3/tools/ete_evol.py

```diff
@@ -688,10 +688,10 @@ def clean_exit(a, b):
 
 
 def load_model(model_name, tree, path, **kwargs):
-    model_obj = Model(model_name, tree, **kwargs)
+    model_obj = Model(model_name.decode('utf-8'), tree, **kwargs)
     setattr(model_obj, 'run', run)
     try:
-        tree.link_to_evol_model(path, model_obj)
+        tree.link_to_evol_model(path.decode('utf-8'), model_obj)
```

Comment:
Fingers crossed again. Almost certain fail on py2 if user has a non-utf8 environment.
Don't know how to do better with py2. Probably unfixable...

Py2 'dead' in 2 years so... time to start aiming at py3 only? It will ease some of the pain.

Reply:
thanks @unode for the feedback!
the tests passes for both py2 and py3 (run_test and test_all)... don't ask me how... I am just still shocked about the fact that it needs all this crazy encoding-decoding... Also I don't understand how it was passing before (none of us were running the tests correctly... recent changes in python3...?)
About forgetting about python2, I installed py3 only for this purpose, so if you ask me... it's still a bit early :P

Reply:
You'll be an actor on "The Walking Python" :smiling_imp: